scripts/scil_denoise_sf.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Script to compute per-vertices hemisphere-aware (asymmetric) Gaussian
+filtering of spherical functions (SF) given an array of spherical harmonics
+(SH) coefficients.
+
+The resulting SF can be expressed using a full SH basis (to keep the
+asymmetry resulting from the filtering) or a symmetric SH basis (where the
+effect of the filtering is a simple denoising).
```

Comment:
Can you add a word on the sharpness and sigma parameters please?
i.e. what does it do, the higher the more sharp? e.g. sharpness = 3 is an agressive sharpening. 0.5 is very conservative. Recommended to be in the range of [X, Y] for good behavior. 

Also, a word on your mask options... they are a bit unusual to me... not quite sure what you are outputting.

Reply:
Done and removed the mask output as it isn't a result of the filtering.